server/routes/auth.js

```diff
@@ -1,20 +1,23 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { validateRegister, validateLogin } = require('../validate');
-const protect = require('../middleware/auth');
+const { validateRegister, validateLogin } = require("../validate");
+const protect = require("../middleware/auth");
 const {
   registerUser,
   loginUser,
   loadUser,
   logoutUser,
-} = require('../controllers/auth');
+  pingLogout
+} = require("../controllers/auth");
 
-router.route('/register').post(validateRegister, registerUser);
+router.route("/register").post(validateRegister, registerUser);
 
-router.route('/login').post(validateLogin, loginUser);
+router.route("/login").post(validateLogin, loginUser);
 
-router.route('/user').get(protect, loadUser);
+router.route("/user").get(protect, loadUser);
 
-router.route('/logout').get(logoutUser);
+router.route("/logout").get(logoutUser);
```

Comment:
These file changes are due to your linters setup. Configure your code editor to use the linters configs which are already included in the client folder of your app.

Reply:
Applied